local-tests/test_multiple_restarts.py

```diff
@@ -24,13 +24,15 @@ def printt(s): print(ctime() + ' | ' + s)
 
 chain.set_flags('no-mdns',
                 port=Seq(30334),
+                validator_port=Seq(3043),
```

Comment:
why not default 30343?

Reply:
Typo. I blame python, based on my personal prejudice.

Fixed.